spec/controllers/items_controller_spec.rb

```diff
@@ -0,0 +1,12 @@
+require 'rails_helper'
+
+RSpec.describe ItemsController, type: :controller do
+
+  describe "POST #create" do
+    it "redirects to user#show page" do
+      get :create
```

Comment:
line 7 in this test is trying to build the url that will be visited.  ask yourself what the url is supposed to look like (you can try `rake routes` if you can't figure it out).  (hint: there are some parameters missing)